codespell_lib/data/dictionary.txt

```diff
@@ -860,7 +861,7 @@ afficionados->aficionados
 affilate->affiliate
 affilates->affiliates
 affilliate->affiliate
-affinitied->affinitized
```

Comment:
Can you add affinitized as a typo too.

Reply:
I'm not sure, what to suggest as a fix.

Reply:
I added affinitized as typo and rebased from master.

Reply:
I learned in a different mergerequest, that rebase and squash shouldn't be done. I will stop doing in the future, if it helps.

Reply:
Thanks, I think the rebase is okay, it's the squash that trashes all the review comments and makes the whole review process a lot harder.